src/Symfony/Component/Messenger/Command/FailedMessagesPurgeCommand.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Messenger\Command;
+
+use Symfony\Component\Console\Exception\RuntimeException;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\ConsoleOutputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Messenger\Transport\Receiver\ListableReceiverInterface;
+use Symfony\Component\Messenger\Transport\Receiver\ReceiverInterface;
+
+/**
+ * @author Ryan Weaver <ryan@symfonycasts.com>
+ *
+ * @experimental in 4.3
+ */
+class FailedMessagesPurgeCommand extends AbstractFailedMessagesCommand
```

Comment:
I was expecting this command to empty the entire failure queue and not to remove a single message.
Not because it's better, just because of the word `purge` which makes me think of the `purge queue` button on SQS. Could `reject` be more inline? Just wondering

Reply:
+1, though I renamed to `remove`. `reject` is more technically correct, but we were already using the word "remove" in all the text communicated to the user around this command.